dist/imgix-core-js.cjs.js

```diff
@@ -149,7 +149,7 @@ function _nonIterableRest() {
 }
 
 // package version used in the ix-lib parameter
-var VERSION = '2.3.2'; // regex pattern used to determine if a domain is valid
+var VERSION = '3.0.0-beta.0'; // regex pattern used to determine if a domain is valid
 
 var DOMAIN_REGEX = /^(?:[a-z\d\-_]{1,62}\.){0,125}(?:[a-z\d](?:\-(?=\-*[a-z\d])|[a-z]|\d){0,62}\.)[a-z\d]{1,63}$/i; // minimum generated srcset width
 
```

Comment:
I couldn't help but notice that the comments here (L152-L160) are all shifted up by one. I'm sure that it's being outputted this way by rollup so there might not be anything actionable on our end.